docs/common/install_domain_certificate.md

```diff
@@ -0,0 +1,43 @@
+# Manual installation of your domain certificate
+
+## About
+This document explains how to manually install your domain certificate in your virtual machine or local environment. 
+
+## Installing the certificate
+Copy the [pfx certificate mentioned in the prerequisites](../prerequisites/readme.md) to your virtual machine or local environment and double click on it to run the certificate import wizard.

```

Comment:
**nitpick (blocking)**: Fix this broken link ../prerequisites/readme.md. I think it is ../prerequisites/README.md